tests/Makefile

```diff
@@ -1,12 +1,19 @@
+ifneq ($(KERNELRELEASE),)
 obj-m := testmodule.o
 testmodule-objs := $(TEST_NAME).rust.o
-KDIR ?= /lib/modules/$(shell uname -r)/build
+
+$(src)/target/x86_64-linux-kernel/debug/lib%.a: $(src)/$(TEST_PATH)/Cargo.toml $(wildcard $(src)/$(TEST_PATH)/src/*.rs)
+	cd $(src)/$(TEST_PATH); env -u MAKE -u MAKEFLAGS RUSTFLAGS="-Dwarnings" CARGO_TARGET_DIR=../target cargo build -Z build-std=core,alloc --target=x86_64-linux-kernel
```

Comment:
can / should we pass `-Dwarnings` from the Python script? (no strong opinion, just want to make sure that it still works to do so....)

Reply:
Yes, I think this is a good idea, I'll do that.